openpype/tools/project_manager/project_manager/window.py

```diff
@@ -28,6 +32,9 @@ def __init__(self, parent=None):
         self._password_dialog = None
         self._user_passed = False
 
+        # keep track of the current project PM is viewing
+        self._current_project = None
```

Comment:
I used current project variable too but combobox may have different race conditions of what is when triggered and what value it has when it's triggered. It happened to me that project name showed in combobox was not the name shown in view. I highly recommend to not use this at least not in this PR.

Reply:
BTW: I have solution which worked 100% and hides `< Select Project >` item.

Reply:
> I used current project variable too but combobox may have different race conditions of what is when triggered and what value it has when it's triggered. 

I don't really understand the concern about combobox ambiguity, 

> It happened to me that project name showed in combobox was not the name shown in view. I highly recommend to not use this at least not in this PR.

I created couple of testing projects, and for them the project names in combobox are identical to what show up in the view.. unless you meant the '<Select Project>' item. 

When calling OpenPype API to create the folders I'll need the project name as in put. If having a private variable is not desirable here, what's a better solution? Per your next comment, could you point to that solution so I can reference it?  😄 


P.s.  Just occurred to me: maybe I should build in checks to detect when we try to re-create these static folders for a project, and abort.

Reply:
The issue is that you're storing combobox text (project name) but there can be different order of processed callbacks and value stored under `_current_project` may not be the same as value selected in combobox and since you've changed that `_current_project` value is source of selected project it can be an issue. But probably don't worry about it in this PR. It's edge case and like I've wrote I have more safe solution which will be next PR.